apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -174,18 +176,6 @@ const aiTutorSlice = createSlice({
       state.showSuggestedPrompts = action.payload;
     },
   },
-  extraReducers: builder => {
```

Comment:
After the response was received, there was a slight delay in setting `isWaitingForChatResponse` so that the animation GIF was shown immediately after the latest assistant message was displayed. Setting `isWaitingForChatResponse` right after receiving the chat response resolves this momentary display of the gif.